subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -198,6 +198,9 @@ JVM auto-detection knows how to work with:
 
 Among the set of all detected JRE/JDK installations, one will be picked according to the <<sec:precedence,Toolchain Precedence Rules>>.
 
+If you are configuring <<sec:custom_loc>>, note that installations that are non-existing or without a
```

Comment:
I would put this in a `NOTE` for better visibility.

Also, this is not only for custom locations, it applies to auto detected ones as well.